docs/code/KeyValues.md

```diff
@@ -1,149 +1,155 @@
 # Key Values
 
-Most commonly, key-values are used to encode items where it is convenient or
-efficient to represent values using numbers, but you want to maintain the
-logical "idea" that these numbers are keys indexing some underlying, implicit
-set of values, in a way more explicit than simply mapping to a number would
-allow you to do.
+Most commonly, in key-valued data, each value takes one of a limited number of
+distinct values. We might view them as being the enumeration into a set. They
+are represented in memory using unsigned integers. Most commonly this is
+`uint`, but `byte`, `ushort`, and `ulong` are usable as well.
 
 A more formal description of key values and types is
 [here](IDataViewTypeSystem.md#key-types). *This* document's motivation is less
 to describe what key types and values are, and more to instead describe why
 key types are necessary and helpful things to have. Necessarily, this document,
 is more anecdotal in its descriptions to motivate its content.
```

Comment:
Here we use `key type` to refer to `KeyDataViewType`, I also notice that we don't mention `KeyDataVeiwType` anywhere in this document.
Would it be useful to make the reference explicit and mention the actual name of the type?

Reply:
Ah, you're right, I never actually mention that, and I totally should. Let me see...

Let's see... I'll add it as an expansion on the second paragraph of `The Representation of Keys`.